elastic/kibana
# [SecuritySolution] Add SecuritySolution sample data

## Summary

- Test environment: https://p.elstc.co/paste/wPDsxkBY#4icff2HWTA784+qCm7q1p-Ct3SVF4WhiM7VdZojwjpJ

**Steps to verify**:
`Try sample data > Security Solution data > Add data`
- It should populate sample data: auditbeat, logs, and alerts
- It shouldn't affect existing auditbeat, logs or alerts data
- It should install relevant saved objects and ingest pipelines


`Try sample data > Security Solution data > Remove data`
- It should remove sample data
- It shouldn't affect existing auditbeat, logs or alerts data
- It should remove relevant saved objects and ingest pipelines

- [x] Add Security Solution sample data:

<img width="2541" alt="Screenshot 2023-09-20 at 20 51 03" src="https://github.com/elastic/kibana/assets/6295984/29956491-12c0-475a-bd84-1f4b8f69b54f">




- Install sample data



Uploading Screen Recording 2023-09-20 at 20.48.28.mov…




- Remove sample data




https://github.com/elastic/kibana/assets/6295984/cebf9f75-5594-402e-997d-40e8bff7df01





- [x] Update fields with ingest pipeline:
<img width="2544" alt="pipeline" src="https://github.com/elastic/kibana/assets/6295984/09dc5c80-f20e-4ca4-a699-505d7e2f79cc">

- [x] Create `kibana_sample_data_securitysolution_alerts` index
- [x] Add sample alert index to security solution alerts alias (and the alerts alias should not be deleted when sample data removed)

<img width="2558" alt="Screenshot 2023-09-14 at 16 11 26" src="https://github.com/elastic/kibana/assets/6295984/2e5ee119-414a-4fc1-a552-fa1f0a811322">

- [x] Create `kibana_sample_data_securitysolution_auditbeat` index
- [x] Add Auditbeat sample index to an alias that matches auditbeat-* pattern:

<img width="2557" alt="Screenshot 2023-09-14 at 16 12 57" src="https://github.com/elastic/kibana/assets/6295984/c9baa324-913b-40e8-b34d-17204180123b">



### Checklist

Delete any items that are not applicable to this PR.
